x/interchainstaking/keeper/callbacks.go

```diff
@@ -253,6 +255,62 @@ func DepositIntervalCallback(k *Keeper, ctx sdk.Context, args []byte, query icqt
 	return nil
 }
 
+func SigningInfoCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes.Query) error {
```

Comment:
Could you add a test for this?

Normal keeperSuite setup, add a consensus address for validatorSet[0] and then pass a SigningInfo into the callback handler and assert the validator record is marked as tombstoned.